### PR TITLE
#21 Add EventStore

### PR DESCRIPTION
**Pull request by karol-be**
* DocumentAddedEvent on source document upload
* Source documents are queryable

### CODE REVIEW

**Comment by karol-be** [comment]
BTW. EventStore management panel can be accessed at localhost:2113,
will add this to readme after merging

File: server/CommandHandler/Handlers/IResult.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
+﻿using System.Threading.Tasks;
+
+namespace CommandHandler.Handlers
+{
+    public record Result
+    {
+        public bool Succeeded { get; init; }
+        public string ErrorMessage { get; init; }
+
+        public static Result Success() => new() { Succeeded = false };
```
Comment (therrlupardo) [inline server/CommandHandler/Handlers/IResult.cs:10]: Shouldn't it be `true`?

File: server/EventsFacade/Utilities/RegisterEventsModule.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
+﻿using EventStore.Client;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace EventsFacade.Utilities
+{
+    public static class RegisterEventsModule
+    {
+        private const string EventStoreConnectionString = "esdb://eventstore.db:2113?tls=false";
```
Comment (therrlupardo) [inline server/EventsFacade/Utilities/RegisterEventsModule.cs]: Can it be moved to some configuration file?

**Review by therrlupardo — APPROVED** [review]

File: server/EventsFacade/Utilities/RegisterEventsModule.cs (hunk 2 above)
Comment (karol-be) [reply server/EventsFacade/Utilities/RegisterEventsModule.cs]: Sure 👍🏽

**Review by zemdom — APPROVED** [review]

**Pull request merged**